k8s/overlays/example/jitsi-common.env

```diff
@@ -3,3 +3,21 @@ PUBLIC_URL=https://jitsi.example.com
 
 # Internal XMPP domain.
 XMPP_DOMAIN=jitsi.example.com
+
+## Authentication
```

Comment:
I'm not sure authentication should be set in the example because it would require a gateway to work?